client/web/src/enterprise/batches/preview/CreateUpdateBatchChangeAlert.tsx

```diff
@@ -66,11 +70,15 @@ export const CreateUpdateBatchChangeAlert: React.FunctionComponent<CreateUpdateB
                     {batchChange && (
                         <>
                             This operation will update the existing batch change{' '}
-                            <Link to={batchChange.url}>{batchChange.name}</Link>. Click 'Apply spec' or run{' '}
-                            <code>src batch apply</code> against your batch spec to update the batch change and perform
-                            the indicated action on each changeset.
+                            <Link to={batchChange.url}>{batchChange.name}</Link>
                         </>
-                    )}
+                    ) : (
+                        'Review the proposed changesets below.'
+                    )}{' '}
+                    Click 'Apply' or run <code>src batch apply</code> against your batch spec to{' '}
+                    {batchChange ? 'update' : 'create'} the batch change and perform the indicated action on each
+                    changeset. Select a changeset and modify the action to customize the publish status of each or all
```

Comment:
After the discussion in #23966:

```suggestion
                    changeset. Select a changeset and modify the action to customize the publication state of each or all
```

Reply:
Ah yep, good catch. I totally messed up the rebase here since there was a merge conflict with a new change from `main` and screwed up the ternary. Turns out I dropped the word change too. 🤦‍♀️